AAUnlimited/Functions/Shared/Triggers/Actions.cpp

```diff
@@ -1496,6 +1496,21 @@ namespace Shared {
 			}
 		}
 
+		// string text, bool important
+		void Thread::Notification(std::vector<Value>& params) {
+			auto text = params[0].strVal;
+			int important = params[1].bVal ? 2 : 1;
+
+			int neededSize = WideCharToMultiByte(CP_UTF8, 0,	// Wstr to str
+				&text->c_str()[0], (int)text->size(), NULL, 0, NULL, NULL);
+			std::string strUTF8(neededSize, '\0');
+			WideCharToMultiByte(CP_UTF8, 0, &text->c_str()[0], (int)text->size(), 
+				&strUTF8[0], neededSize, NULL, NULL);
+			char *text_char = new char[strUTF8.size() + 1];		// From string to char
```

Comment:
why not use
```
auto text_char = General::CastToString(text ->c_str()).c_str();
```
?

Reply:
That advice is wrong. You don't use internal data (c_str) of a temporal object (CastToString return isn't assigned to anything in this scope)

Reply:
Ok, i'll using wstring then.